src/weekly-notes/action.yml

```diff
@@ -2,14 +2,16 @@ name: 'Open weekly issue'
 description: 'Create new release issue when the last one was closed'
 inputs:
   token:
-    description: 'GITHUB_TOKEN or a repo scoped PAT.'
+    description: 'GITHUB_TOKEN or a repo scoped PAT'
     default: ${{ github.token }}
   template-path:
     description: 'Relative path to the weekly template'
     required: true
     default: '.github/ISSUE_TEMPLATE/WEEKLY_NOTE.md'
   moderators-path:
     description: 'Relative path to the list of moderators'
+  include-community-worker:
```

Comment:
```suggestion
  assign-community-worker:
```

Reply:
Nevermind this comment. Maybe worth simplifiying this to just `community-worker=true|false`

Reply:
Simplified